spec/support/fixtures/vcr_cassettes/parcels_request/with_updated_after.yml

```diff
@@ -22,89 +22,34 @@ http_interactions:
-        50","address_2":"","address_divided":{"street":"Strobelallee","house_number":"50"},"city":"Dortmund","company_name":"","country":{"iso_2":"DE","iso_3":"DEU","name":"Germany"},"data":{},"date_created":"09-09-2021
-        09:28:03","date_announced":"09-09-2021 09:28:03","date_updated":"12-09-2021
-        20:02:00","email":"","name":"Test Jon","postal_code":"44139","reference":"0","shipment":{"id":8,"name":"Unstamped
-        letter"},"status":{"id":11,"message":"Delivered"},"to_service_point":null,"telephone":"","tracking_number":"SCCWF28TQ6BP","weight":"1.000","label":{"normal_printer":["https://panel.sendcloud.sc/api/v2/labels/normal_printer/127396222?start_from=0","https://panel.sendcloud.sc/api/v2/labels/normal_printer/127396222?start_from=1","https://panel.sendcloud.sc/api/v2/labels/normal_printer/127396222?start_from=2","https://panel.sendcloud.sc/api/v2/labels/normal_printer/127396222?start_from=3"],"label_printer":"https://panel.sendcloud.sc/api/v2/labels/label_printer/127396222"},"customs_declaration":{},"order_number":"","insured_value":0,"total_insured_value":0,"to_state":null,"customs_invoice_nr":"","customs_shipment_type":null,"parcel_items":[],"documents":[{"type":"label","size":"a6","link":"https://panel.sendcloud.sc/api/v2/parcels/127396222/documents/label"}],"type":"letter","shipment_uuid":null,"shipping_method":8,"external_order_id":"127396222","external_shipment_id":"","external_reference":"13477287","is_return":false,"note":"","to_post_number":"","total_order_value":null,"total_order_value_currency":null,"colli_tracking_number":"","colli_uuid":"14d84d50-13fb-4c62-b7c1-25e3d6e6f366","collo_nr":0,"collo_count":1,"awb_tracking_number":null,"box_number":null,"length":null,"width":null,"height":null,"shipping_method_checkout_name":null,"carrier":{"code":"sendcloud"},"tracking_url":"https://tracking.sendcloud.sc/forward?carrier=sendcloud&code=SCCWF28TQ6BP&destination=DE&lang=en-gb&source=NL&type=letter&verification=44139&created_at=2021-09-09"}]}'
-  recorded_at: Wed, 22 Sep 2021 15:08:14 GMT
-recorded_with: VCR 6.0.0
+      string: !binary |-
```

Comment:
I saw this changed for various recordings when I tried to "quickly" (😅) re-record these during the original support issue. I wasn't sure why the response was suddenly being recorded as binary data - had a hunch that it's VCR not correctly handling a compressed response but didn't look into it any further than that. 

Ideally the responses is the recordings should be human readable, so it would be worth looking at whether we can get around this somehow.

also... for our own sanity... do you think it's worth specifying a smaller page size on these parcels requests if their API supports it?

Reply:
That's a good point, I've added some configuration to handle compressed responses in https://github.com/BloomAndWild/sendcloud/pull/30/commits/1c72d89aaab0c6a78ed458d6dfa637245d4cb2f6

It doesn't look like there's an option to specify the page size so I think we're stuck with 100 😬

Reply:
Nice 🙌 thanks for fixing this!